.gitignore

```diff
@@ -1,2 +1,5 @@
 /node_modules
 /.vscode
+/src/typed-error.d.ts
+/src/typed-error.js
+/src/typed-error.js.map
```

Comment:
You need to add these to the package.json `files` I think, or create an `npmignore`, because by default gitignore is also used to exclude files from publishing. https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package

Reply:
You might be right, but I realised after PRing this that my local master had been on 2.x instead of 3.x.. so I've had to redo most of it